### PR TITLE
ICP-7323 ICP-7309 ICP-7308 ICP-7289 ICP-7282 ICP-7288 Stelpro Ki zigb…

### PR DESCRIPTION
**Pull request by greens**
…ee issues

* Missing a unit when reporting a setpoint change
* Possibly preventing operating state changes from being propagated
* Device seems to use a different C->F conversion than us

### CODE REVIEW

File: devicetypes/stelpro/stelpro-ki-zigbee-thermostat.src/stelpro-ki-zigbee-thermostat.groovy [hunk 1]
```diff
@@ -315,10 +315,6 @@ def parse(String description) {
 				} else {
 					map.value = "heating"
 				}
-
-				if (settings.heatdetails == "No") {
-					map.displayed = false
```
Comment (dkirker) [inline devicetypes/stelpro/stelpro-ki-zigbee-thermostat.src/stelpro-ki-zigbee-thermostat.groovy:320]: This should only affect the event history, and in Classic. If this is actually preventing events from reaching the event pipeline, then there is a bug in data management.

See https://community.smartthings.com/t/stelpro-releases-the-first-z-wave-plus-line-voltage-thermostat-the-stzw402-stelpro-ki-thermostat/39278/179 for a reference to the purpose of this.

Comment (greens) [reply devicetypes/stelpro/stelpro-ki-zigbee-thermostat.src/stelpro-ki-zigbee-thermostat.groovy:320]: Based on the live logs I was seeing, it wasn't getting switched anyway (but the events were). I was just throwing stuff at the wall.

Comment (greens) [reply devicetypes/stelpro/stelpro-ki-zigbee-thermostat.src/stelpro-ki-zigbee-thermostat.groovy:320]: My guess is it's actually a ui metadata pointing at the wrong href.

Comment (dkirker) [reply devicetypes/stelpro/stelpro-ki-zigbee-thermostat.src/stelpro-ki-zigbee-thermostat.groovy:320]: Turns out that the device was being configured to not report changes more frequently than 5 minutes, as well as the cadence of the duty cycles and the code that handles passing off the Zigbee messages. The ticket(s) include details.

**Pull request closed without merging**